### ISSUE TITLE
[A11y] 2.5.7 Dragging Movements - Interactions that Uses Dragging Movement Cannot be Achieved Without Dragging

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1761]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/46 for details
